docs/index.html

```diff
@@ -1,79 +1,112 @@
-<!doctype html>
-<html lang="en">
-    <head>
-        <meta charset="utf-8">
-        <title>harp.gl</title>
-        <meta name="description" content="Landing page for harp.gl">
-        <meta name="viewport" content="width=device-width">
-        <link rel="stylesheet" href="./css/style.css">
-        <script   src="https://code.jquery.com/jquery-3.4.1.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
+<html>
+   <head>
+      <title>harp.gl</title>
+      <link href="https://fonts.googleapis.com/css?family=Fira+Sans:400,500,600&display=swap" rel="stylesheet">
+      <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+      <meta name="viewport" content="width=device-width,initial-scale=1">
+      <link rel="stylesheet" type="text/css" href="css/index.css">
+      <link rel="shortcut icon" href="img/favicon.ico">
+   </head>
+   <body>
+      <header>
+         <iframe src="https://dbabbs.github.io/harp-temp-site/dist/"></iframe>
```

Comment:
I guess that the src of this iframe should not be that?

Reply:
I had thought about this too, but where else should we host? Within this repository inside of `docs`?

Reply:
I think we can host it directly in our `gh-pages`. Same way we host right now the `releases.json` resource that is only needed for this site.